reviewrot/pagurestack.py

```diff
@@ -72,14 +73,19 @@ def request_reviews(self, user_name, repo_name=None, state_=None,
         res_ = []
         for res in response['requests']:
             # if namespace exists in response
-            try:
-                repo_reference = os.path.join(res['project']['namespace'],
-                                              res['project']['name'])
-            except:
+
+            if res['project']['namespace']:
+                repo_reference = res['project']['namespace'] + '/' + \
+                                 res['project']['name']
+            else:
                 repo_reference = res['project']['name']
+
+            last_comment = self.get_last_comment(res)
+
             # format pull request url
-            url = os.path.join('https://pagure.io/', repo_reference,
-                               'pull-request', str(res['id']))
+            url = urllib.parse.urljoin(
+                'https://pagure.io/',
+                repo_reference + '/' + 'pull-request/', str(res['id']))
```

Comment:
not sure if this is a good way how to approach concatenation of address.. you can use urllib.parse.urljoin twice or more

Reply:
or use one string it this case